src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -128,6 +74,70 @@ export const writePackageTransform = (options: NgPackagrOptions) =>
     return graph;
   });
 
+async function copyAssets(
+  graph: BuildGraph,
+  entryPointNode: EntryPointNode,
+  ngPackageNode: PackageNode,
+): Promise<void> {
+  const ngPackage = ngPackageNode.data;
+
+  const pathsIgnored: string[] = [
+    '.gitkeep',
+    '**/.DS_Store',
+    '**/Thumbs.db',
+    '**/node_modules/**',
+    `${ngPackage.dest}/**`,
+  ];
+
+  const assets: AssetEntry[] = [];
+
+  for (const item of ngPackage.assets.concat('LICENSE', '**/README.md')) {
+    let asset: AssetEntry;
+    if (typeof item == 'object') {
+      asset = item;
+      asset.glob = path.join(ngPackage.src, asset.input, asset.glob);
+    } else {
+      const glob = path.join(ngPackage.src, item);
+      const isDir = await stat(glob)
+        .then(stats => stats.isDirectory())
+        .catch(() => false);
+      asset = { glob: isDir ? path.join(glob, '**/*') : glob, input: '.', output: '/' };
+    }
+
+    asset.input = path.join(ngPackage.src, asset.input);
+    asset.output = path.join(ngPackage.dest, asset.output);
+    asset.ignore = [...(asset.ignore ?? []), ...pathsIgnored].map(glob => path.join(asset.input, glob));
```

Comment:
There is no need to join these ignored with the input paths. 

```suggestion
    asset.ignore = [...(asset.ignore ?? []), ...pathsIgnored];
```